renpy.el

```diff
@@ -2059,7 +2143,15 @@ with skeleton expansions for compound statement templates.
   ;; Install the capf function.
   (when renpy-setup-completion
     (add-hook 'completion-at-point-functions
-	      #'renpy-completion-at-point nil t)))
+	      #'renpy-completion-at-point nil t))
+
+  ;; Setup the Flymake backend.
+  (add-hook 'flymake-diagnostic-functions #'renpy--flymake-backend nil t)
+  ;; NOTE: No need to check buffer state at all as the linter works with all
```

Comment:
I would remove the "NOTE: " part of the comment because that doesn't really add anything to the description of why the value is being set.